galley/pkg/fs/fssource.go

```diff
@@ -39,7 +39,7 @@ var supportedExtensions = map[string]bool{
 	".yaml": true,
 	".yml":  true,
 }
-var scope = log.RegisterScope("file-source", "Source for File System", 0)
+var scope = log.RegisterScope("runtime", "Galley runtime", 0)
```

Comment:
This is not runtime. This is a sister package to kube.

Reply:
We can make this consistent with the "kube", one as they are at the same level functionally. How about "fs"? It makes both scopes named as their package name.